content/en/docs/Concepts/DTO.md

```diff
@@ -1,11 +1,122 @@
 ---
-title: "Authentication"
+title: "DTO - Data Transfer Objects"
 linkTitle: "DTOs (Data Transfer Objects)"
 weight: 35
 description: >
-  Lead in description for DTOs
+  Learn about data transfer objects (aka ViewModels in Asp.net MVC).
 ---
 
-## Overview
+## What are they?
+
+A data transfer object is a class that’s used for public display of another object in the app. DTO makes a class more presentable by hiding sensitive data.
+
+## Benefits of using them over raw entities?
+
+A layer of DTOs isolates the domain model from the presentation, resulting in both loose coupling and optimized data transfer. In loose coupling, you can change one service without changing the other by reducing interdependencies. If DTOs are used, a change in the requirements that forces a move to a different amount of data doesn't have any impact on the service layer or even the domain.
+
+## How does Asp.Net core prevent overposting attacks by using field binding?
+
+### - What is overposting?
+
+Overposting is a type of cyber attack that targets websites. More specifically, overposting targets models that expect an input from a user, but have other variables that are intentionally hidden from the user. For example, if a model has a “IsAdmin” variable that limits what a user can and cannot do, a hacker who knows this information can manually add an extra field to an input that would allow them to change the “IsAdmin” variable.
+
+### - How is this prevented?
+
+Take a look at an example set of code \(provided by [Scott Hanselmen](https://www.hanselman.com/blog/aspnet-overpostingmass-assignment-model-binding-security)\):
```

Comment:
It wasn't provided to us, but rather we took it from him, so just change that wording and that'll be good!

Reply:
Remove the parenthesis for "provided by Scott Hanselmen". Since we aren't doing in-text citations, references like that should be outside of the parenthesis since they are important to the sentence.